src/overlays/actors/ovl_En_Ta/z_en_ta.c

```diff
@@ -10,26 +10,52 @@
 
 #define FLAGS (ACTOR_FLAG_0 | ACTOR_FLAG_3)
 
+#define TALON_STATE_FLAG_TRACKING_PLAYER (1 << 0)
+#define TALON_STATE_FLAG_GIVING_MILK_REFILL (1 << 1)
+#define TALON_STATE_FLAG_SUPPRESS_BLINK (1 << 2)
+#define TALON_STATE_FLAG_3 (1 << 3)
+#define TALON_STATE_FLAG_ANIMATION_FINISHED (1 << 4)
+#define TALON_STATE_FLAG_CUCCO_GAME_START_EVENT_TRIGGERED (1 << 5)
+#define TALON_STATE_FLAG_UNUSED (1 << 6)
+#define TALON_STATE_FLAG_FLOOR_CAMERA_ACTIVE (1 << 7)
+#define TALON_STATE_FLAG_RAISING_HANDS (1 << 8)
+#define TALON_STATE_FLAG_RESTORE_BGM_ON_DESTROY (1 << 9)
+
+#define TALON_FACE_REACTION_SET 24
+
+#define EVENTINF_CUCCO_GAME_FINISHED EVENTINF_HORSES_0A
+#define EVENTINF_CUCCO_GAME_WON EVENTINF_HORSES_08
```

Comment:
You can just rename `EVENTINF_HORSES_08` and `EVENTINF_HORSES_0A`

The idea with the common `horses_` part is because all the `0x00-0x0F` flags are logically tied by statements like `gSaveContext.eventInf[EVENTINF_HORSES_INDEX] = 0;` which clears all the flags at once (not sure yet what the purpose is)

So idk maybe `EVENTINF_HORSES_CUCCO_GAME_WON` and such :smile:

Reply:
Those flags are used in other minigames, too: 08 in Gerudo archery and 0A in Malon's race. So renaming them globally with these names would not be that great as they are also used for other purposes

Reply:
Oh, I didn't notice that. Then I wouldn't name the flags at all yet, since these bits pertain to the same memory regardless of the actor they're used in I don't think different names should be used to designate them
but you could add a comment on the `EVENTINF_HORSES_08`/`0A` defines about their use for Talon's cucco minigame

Reply:
I added the comments (and tried to see how the flags are used in the other minigames), but still left these defines in place until the horse flags are renamed

Reply:
I think at the very least, if we are going with this strategy of giving these flags aliases depending on context, they should be defined right next to the original definition of the flag, as opposed to locally in the actor. That makes it pretty hard to track things down imo.

Reply:
Personally I prefer the aliases, because the alternatives (naming the multi-purpose flags something like `EVENTINF_TALON_CUCCO_GAME_WON_OR_GERUDO_ARCHERY_WHATEVER` or just using a generic name with comments explaining its use) are worse and more difficult to read.

I will move these aliases to z64save.h. Or I can just revert these to the original names if the aliases are not something that you want to introduce right now

Reply:
Our handling and even the naming scheme of these flags is not something thats very established yet. So it will take time for us to settle on something we like

It has its pros and its cons, the aliases make the code nicer to read, but also make it easier for one to not realize that the flags are shared (and there are many bugs in the game because of accidental shared flag usage, so it would be nice to help prevent future users of the codebase from making the same mistakes)

But yea im fine with an alias for now as long as they are defined in a centralized place